src/common/components/vanilla/table-interactive/table.css

```diff
@@ -37,6 +39,27 @@ $table-row-border-color: $dark-grey;
   vertical-align: middle;
 }
 
+.header, .data {
+  composes: cell;
+  display: table-cell;
+  float: left;
+}
+
+.dataLink {
+  composes: data;
+  color: $cool-grey;
+}
+
+a.dataLink:hover,
+a.dataLink:focus,
+a.dataLink.active {
```

Comment:
I don't like too much putting tag name in CSS here, but it's the easiest thing to do to distinguish cell that is a link from the one that is not without introducing another class name (that will serve exactly the same purpose).